plugin/trino-exasol/src/main/java/io/trino/plugin/exasol/mapping/ColumnMapper.java

```diff
@@ -0,0 +1,60 @@
+package io.trino.plugin.exasol.mapping;
+
+import static java.lang.String.format;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.temporal.ChronoField;
+
+import io.trino.plugin.jdbc.ColumnMapping;
+import io.trino.plugin.jdbc.JdbcTypeHandle;
+
+/**
+ * Abstract base class for mapping JDBC column types to Trino {@link ColumnMapping}s.
+ * <p>
+ * Subclasses should implement the {@link #map(JdbcTypeHandle)} method to return
+ * a suitable mapping for specific database column types (e.g., Exasol-specific mappings).
+ */
+public abstract class ColumnMapper {
```

Comment:
Introducing mapper class is overkill. Move all new methods to `ExasolClient`. 
Most javadoc look unhelpful. Please remove them.